pkg/util/allocationfilterutil/queryfilters.go

```diff
@@ -416,3 +416,220 @@ func filterV1DoubleValueFromList(rawFilterValuesUnsplit []string, filterField ku
 
 	return filter
 }
+
+type FilterV1 struct {
+	FilterClusters        []string `json:"filterClusters"`
+	FilterNodes           []string `json:"filterNodes"`
+	FilterNamespaces      []string `json:"filterNamespaces"`
+	FilterControllerKinds []string `json:"filterControllerKinds"`
+	FilterControllers     []string `json:"filterControllers"`
+	FilterPods            []string `json:"filterPods"`
+	FilterContainers      []string `json:"filterContainers"`
+	FilterDepartments     []string `json:"filterDepartments"`
+	FilterEnvironments    []string `json:"filterEnvironments"`
+	FilterOwners          []string `json:"filterOwners"`
+	FilterProducts        []string `json:"filterProducts"`
+	FilterTeams           []string `json:"filterTeams"`
+	FilterAnnotations     []string `json:"filterAnnotations"`
+	FilterLabels          []string `json:"filterLabels"`
+	FilterServices        []string `json:"filterServices"`
+}
+
+func ParseAllocationFilterV1(filters FilterV1, labelConfig *kubecost.LabelConfig, clusterMap clusters.ClusterMap) kubecost.AllocationFilter {
```

Comment:
this function is copy / paste. I plan to refactor the existing implementation into a transform and use this for the logic, to remove duplication.